test_bench.py

```diff
@@ -22,7 +22,7 @@
 def pytest_generate_tests(metafunc):
     # This is where the list of models to test can be configured
     # e.g. by using info in metafunc.config
-    devices = ['cpu', 'cuda']
+    devices = ['cpu', 'cuda', 'mps']
```

Comment:
Can you please add `mps` device only when it is available? Thanks!

Reply:
Done.